src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's Block in the address book
```

Comment:
:+1: for including the header comment :-)
may be needs a small rewording.
Block belongs to the address; not necessarily address book, no?